docs/source/slurmguide/slurmarray.md

```diff
@@ -1,29 +1,25 @@
 (slurm-arrays)=
-# Slurm Array Jobs and Dependencies
+# Slurm Array Jobs
 
-<> (job-arrays)=
-## Slurm Job Arrays
-Job arrays are a convenient way to submit and manage large numbers of similar jobs quickly. They can process millions of tasks in milliseconds, provided they are within size limits. Job arrays are particularly useful when running similar jobs, such as performing the same analysis with different inputs or parameters.
-
-Using job arrays can save time and reduce the amount of manual work required. Instead of submitting each job individually, you can submit a single job array and let Slurm handle the scheduling of individual jobs. This approach is beneficial if you have limited time or resources, as it allows you to use the cluster's computing power more efficiently by running multiple jobs in parallel.
+Job arrays are a convenient way to submit and manage large numbers of similar jobs quickly. Job arrays are particularly useful when running similar jobs, such as performing the same analysis with different inputs or parameters.
 
 There are several ways to define job arrays, such as specifying the range of indices or providing a list of indices in a file. Slurm also offers various features to manage and track job arrays, such as options to simultaneously suspend, resume, or cancel all jobs in the array.
 
-### Syntax: Job Arrays
+:::{warning}
+50 is the maximum number of jobs allowed to be run at once per user-account.
```

Comment:
Its actually both. If you submit 50 jobs with srun or sbatch (50 manual typings of sbatch or srun) or using a job array (sbatch with some special callouts in it) will submit a series of jobs from 1 submission. Also this might be per partition as well because the gpu and multigpu partition have lower job limits. I never tried it with gpu jobs though.

Reply:
Thanks, that makes sense! I don't remember if there was something I thought should be changed depending on the answer, but it seems ok!